app/config.json

```diff
@@ -0,0 +1,37 @@
+[
+	{
+		"id": 1,
```

Comment:
Don't you think looping through the file & updating is a lot simpler this way? Currently, id property (which I'm binding to the input label) is helping me update a particular input only on blur and also it acts as an index in controller. I understand that the file becomes quite verbose but the coding becomes simpler I guess.